dashboard/legacy/middleware/files_api.rb

```diff
@@ -23,7 +23,7 @@ def max_app_size
   # Can set this to an empty array if we do not want aichat checked for profanity.
   LABS_TO_CHECK_FOR_PROFANITY = DCDO.get('labs_to_check_for_profanity', [])
 
-  BACKPACK_PROGRAM_FILE_TYPES = ['.java', '.py']
+  BACKPACK_PROGRAM_FILE_TYPES = ['.csv', '.java', '.py', '.txt']
```

Comment:
mind adding a comment that these file types are not used in app lab, so they are safe to skip checking profanity for?

Reply:
Updated!